charts/prometheus-snmp-exporter/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: Prometheus SNMP Exporter
 name: prometheus-snmp-exporter
-version: 0.0.6
-appVersion: 0.14.0
+version: 0.0.7
```

Comment:
feature update should raise minor version.
would be 0.1.0 in this case what should have been the first version anyway.

Reply:
@monotek Thanks for the correction, should be fixed now :)

Reply:
Lint already considers this 0.1.0, should I bump to 0.1.1 ?

Reply:
Yes, please...

Reply:
Done